jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    preset: 'ts-jest',
+    testEnvironment: 'node',
+
+    // modulePaths: ['./src/foo.test.ts', './test/src/tests-vault.ts'],
```

Comment:
Should this be commented out?  Should it be removed?

Reply:
Yeah, I think that was something I forgot to remove